src/mscorlib/src/System/AppContext/AppContextDefaultValues.cs

```diff
@@ -28,26 +28,22 @@ private static void ParseTargetFrameworkName(out string identifier, out string p
         {
             string targetFrameworkMoniker = AppDomain.CurrentDomain.SetupInformation.TargetFrameworkName;
 
-            // If we don't have a TFM then we should default to the 4.0 behavior where all quirks are turned on.
             if (!TryParseFrameworkName(targetFrameworkMoniker, out identifier, out version, out profile))
             {
 #if FEATURE_CORECLR
-                if (CompatibilitySwitches.UseLatestBehaviorWhenTFMNotSpecified)
```

Comment:
This is still used to set the legacy [BinaryCompatibility](https://github.com/dotnet/coreclr/blob/775003a4c72f0acc37eab84628fcef541533ba4e/src/mscorlib/src/System/Runtime/Versioning/BinaryCompatibility.cs#L466) switches. I don't think we care about that anymore though.

I will file a separate issue to remove this switch (as it touches quite a few files) and is not really needed right now.